packages/syft/src/syft/service/worker/worker_pool_service.py

```diff
@@ -362,9 +362,7 @@ def get_all(
             return SyftError(message=f"{result.err()}")
         worker_pools: list[WorkerPool] = result.ok()
 
-        res: list[tuple] = []
-        for pool in worker_pools:
-            res.append((pool.name, pool))
+        res = [(pool.name, pool) for pool in worker_pools]
```

Comment:
We're passing it to `DictTuple` so using a generator instead of list is a tiny bit more efficient.
```suggestion
        res = ((pool.name, pool) for pool in worker_pools)
```